src/components/Footer.svelte

```diff
@@ -37,5 +38,8 @@
       <a href="https://github.com/mozilla/glean-dictionary/issues">Issues</a>
     </li>
     <li><a href="https://github.com/mozilla/glean-dictionary">Source</a></li>
+    <li>
+      <span>Glean - revision </span><a href={revLink}>{rev.substring(0, 10)}</a>
```

Comment:
We probably ought to rethink the footer in #294, for now I think just a parenthetical next to "Source" ought to do the trick. Saying "Glean version" is problematic for the same reason as I suggest above. 

```suggestion
    <li><a href="https://github.com/mozilla/glean-dictionary">Source</a>
      <span>(</span><a href={`https://github.com/mozilla/glean-dictionary/tree/${rev}`}>{rev.substring(0, 10)})</a>
```